### ISSUE TITLE
Can't set proxy to ArangoDatabase

### ISSUE THREAD

**Issue by jjchiw** [ra0o0f/arangoclient.net#3]
Hi!

I really like what you're doing here, I'm starting to test the driver and It's awesome!

I just have a little question, why the property `ClientSetting` in `ArangoDatabase` is private? https://github.com/ra0o0f/arangoclient.net/blob/master/ArangoDB.Client/ArangoDatabase.cs#L31

I'm trying to see all the http requests that are made to arangodb using fiddler, so I have to put a proxy, I've changed the Property to public in order to set the fiddler proxy

	ArangoDatabase.ClientSetting = new ClientSetting {
		Proxy = new WebProxy("http://127.0.0.1:8888", false),
	};

	using (ArangoDatabase db = new ArangoDatabase(url: "http://localhost.fiddler:8529", database: "myDatabase"))
	{ ... }

Even is just removing the `private` keyword from that line, would you like a PR?

I would really like to help to make this library better with all the functions that exists in arangodb...

Here is a thread talking about the .net driver for arangodb in the googlegroups https://groups.google.com/forum/#!searchin/arangodb/.net/arangodb/Y1fWIPcE8Zs/mG9YxRvzJnIJ

You should post your driver there. :)

**Comment by ra0o0f** [comment]
hi jjchiw

i appreciate your saying and thanks for testing the driver. it is possible to set the proxy directly, without creating new `ClientSetting`

```csharp
ArangoDatabase.ClientSetting.Proxy = new WebProxy("http://localhost:10099", false);

using (var db = new ArangoDatabase(url: "http://localhost.:8529", database: "myDatabase"))
{ ... }
``` 

the reason is `IsHttpClientInitialiezd` internal property that hold the state if HttpClient is initilized, and creating new `ClientSetting` will erase the state, perhaps moving `IsHttpClientInitialiezd` to another class would be a good idea.

@jjchiw please let me know if you find anything that make this driver better.

**Comment by jjchiw** [comment]
Hi.

Ok, I'll set the proxy that way (I'll close the PR #5). Right now I'm testing the driver and I'm adding some stuff, I added a method to drop the database (PR #7) and fixed the create database (PR #6)

I'll let you know and while I'm testing it, I'll fork the documentation and add some.

:)
